Package.swift

```diff
@@ -15,13 +15,15 @@ let package = Package(
     )
   ],
   dependencies: [
-    .package(url: "https://github.com/wulkano/Aperture", from: "0.2.0")
+    .package(url: "https://github.com/wulkano/Aperture", from: "0.2.0"),
+    .package(url: "https://github.com/apple/swift-argument-parser", from: "0.1.0")
```

Comment:
Again, used `0.1.0` because I can only run on Swift 5.1, but it had everything I needed for this

Reply:
After I'm done with the rest of the comments if you want to pull down and migrate, that'd work too. I don't know if any breaking changes happened. I just can't even fetch 0.2.0 without using Swift tools 5.2, so I couldn't test it

Reply:
We can keep it 5.1 until you can upgrade.